msteams-platform/sbs-meetings-sidepanel.yml

```diff
@@ -222,13 +269,17 @@ items:
     ```bash
     dotnet run
     ```
-    :::image type="content" source="~/assets/images/meetings-side-panel/dotnetruncmd.png" alt-text="Screenshot shows Command Prompt dotnet run with the dotnet run command."::: 
-
+    :::image type="content" source="~/assets/images/meetings-side-panel/dotnetruncmd.png" alt-text="Screenshot shows Command Prompt dotnet run with the dotnet run command.":::
+      
     ---
     
     A webpage appears with a message, **Your bot is ready!**
 
-    :::image type="content" source="~/assets/images/meetings-side-panel/app-is-ready.png" alt-text="Screenshot shows the webpage that displays that your bot is ready."::: 
+    :::image type="content" source="~/assets/images/meetings-side-panel/app-is-ready.png" alt-text="Screenshot shows the webpage that displays that your bot is ready.":::
+     
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Build+and+run+the+services&&author=%40surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-meetings-sidepanel%3Ftabs%3Ddev%252Cvs%26tutorial-step%3D6&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-meetings-sidepanel.yml&documentVersionIndependentId=c1a955d5-85bc-7efd-c991-c705cf18d0b0&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
Let's add 2 buttons

Build and run the service - Visual Studio
Build and run the service - Command line

Reply:
Updated as suggested